tests/ignite/metrics/test_fbeta.py

```diff
@@ -22,7 +22,7 @@ def test_wrong_inputs():
 
     with pytest.raises(ValueError, match=r"Input recall metric should have average=False"):
         r = Recall(average="samples")
-        Fbeta(1.0, recall=r)
+        Fbeta(1.0, recall=r, device=torch.device("cpu"))
```

Comment:
Revert this change and write its own "error raised" check

Reply:
like this?
```python
    with pytest.raises(TypeError, match=r"device\(\) received an invalid combination of arguments"):
        r = Recall(average="samples")
        Fbeta(1.0, recall=r)
```